src/datapool/datapool_shm.c

```diff
@@ -41,3 +41,14 @@ datapool_size(struct datapool *pool)
     return cc_alloc_usable_size(pool);
 }
 
+void
```

Comment:
would appreciate comments to document why empty/unused functions are there

Reply:
Done